.travis.yml

```diff
@@ -127,6 +135,69 @@ script:
 jobs:
         include:
                 - stage: test
+                  compiler: aarch64-linux-gnu-gcc
+                  env: TEST="aarch64-linux-gnu" CROSS="--host=aarch64-linux-gnu --prefix=/usr/aarch64-linux-gnu"
+                  install: true
+                  script:
+                          - mkdir cross
+                          - pushd cross
+                            # one can not include colon in the command or bad things will happen, so mimic it
+                          - env echo -e 'Provides\x3a multiarch-support, debconf, debconf-2.0' > dummy
+                          - equivs-build -a arm64 dummy
+                          - wget
+                            http://ports.ubuntu.com/pool/main/o/openssl/libssl-dev_1.0.1f-1ubuntu2.22_arm64.deb
+                            http://ports.ubuntu.com/pool/main/o/openssl/libssl1.0.0_1.0.1f-1ubuntu2.22_arm64.deb
+                            http://ports.ubuntu.com/pool/main/z/zlib/zlib1g-dev_1.2.8.dfsg-1ubuntu1_arm64.deb
+                            http://ports.ubuntu.com/pool/main/z/zlib/zlib1g_1.2.8.dfsg-1ubuntu1_arm64.deb
```

Comment:
@lumag that also need pkg-config-cross and glib2. I'm trying to do it with dpkg --add-architecture arm64; apt-get install libssl-dev:arm64 lib1g-dev:arm64  but there are some problems that index file is not supported. https://github.com/muvarov/odp/commits/api-next_arm64_merge (click on Travis), On previous change instead of install pkg-config failed arm64 build passed. But I think it will be more clean if we can install everything with apt-get.